src/services/git.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path')
+const ProcessManager = require('../utils/processManager')
+const {validateCommand} = require('../utils/utils')
+const gitRegex = '^git\\s[^&|;]*$'
+
+class GitService {
+
+  constructor () {
+    this.processManager = new ProcessManager()
+    this.sharedFolder = path.resolve(__dirname, '../..', process.env.SHARED_FOLDER)
+  }
+
+  /**
+   * Create and check if command is valid
+   * @param action
+   * @param id
+   * @param key
+   * @param name
+   * @param payload
+   * @param callback
+   */
+  command (action, id, key, name, payload, callback) {
```

Comment:
you don't need `action, id, key, name` in this function, at least you don't use it.
The signature can stay `command (payload, callback)`

Reply:
Currently sharedFolder service needs action, id, key, name and git and all others will use name, payload etc. Will have to check second part when remix-plugin is done. This way both services work.